### PR TITLE
Support external dependency in gazelle

### PR DESCRIPTION
**Pull request by yugui**
c.f. #15

### CODE REVIEW

**Comment by yugui** [comment]
@pmbethe09 This is another step toward #15.  Could you take a look?

File: go/tools/gazelle/rules/resolve_external.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	}
+
+	prefix := r.Root
+	var pkg string
+	if importpath != prefix {
+		pkg = strings.TrimPrefix(importpath, prefix+"/")
+	}
+
+	components := strings.Split(prefix, "/")
+	labels := strings.Split(components[0], ".")
```
Comment (pmbethe09) [inline go/tools/gazelle/rules/resolve_external.go]: why not just call sort.Reverse(labels)

**Comment by pmbethe09** [comment]
LGTM with one comment.

File: go/tools/gazelle/rules/resolve_external.go (hunk 1 above)
Comment (yugui) [inline go/tools/gazelle/rules/resolve_external.go]: `sort.Reverse` is not what we want.  It wraps an ascending comparator and returns a descending comparator.

**Pull request merged**